### PR TITLE
create-root

### PR DESCRIPTION
**Pull request by nafistiham**

### CODE REVIEW

File: src/content/reference/react-dom/client/createRoot.md [hunk 1]
```diff
@@ -29,60 +29,60 @@ const domNode = document.getElementById('root');
...
-`root.render` returns `undefined`.
+`root.render` রিটার্ন করে `undefined`.
 
-#### Caveats {/*root-render-caveats*/}
+#### সতর্কতা {/*root-render-caveats*/}
 
-* The first time you call `root.render`, React will clear all the existing HTML content inside the React root before rendering the React component into it.
+* আপনি যখন প্রথম বারের মত `root.render` কল করবেন, React এর রুটের ভিতরে কম্পোনেন্ট রেন্ডার করবার আগে থাকা সব HTML কনটেন্ট মুছে ফেলবে। 
 
 * If your root's DOM node contains HTML generated by React on the server or during the build, use [`hydrateRoot()`](/reference/react-dom/client/hydrateRoot) instead, which attaches the event handlers to the existing HTML.
```
Comment (moshfiqrony) [inline src/content/reference/react-dom/client/createRoot.md:87]: Bhai ei line ta ki intentionally translate korenni?

File: src/content/reference/react-dom/client/createRoot.md [hunk 2]
```diff
@@ -105,28 +105,28 @@ This is mostly useful if your React root's DOM node (or any of its ancestors) ma
 Calling `root.unmount` will unmount all the components in the root and "detach" React from the root DOM node, including removing any event handlers or state in the tree.
```
Comment (moshfiqrony) [inline src/content/reference/react-dom/client/createRoot.md:105]: Bhai ei line ta ki intentionally translate korenni?

File: src/content/reference/react-dom/client/createRoot.md [hunk 3]
```diff
@@ -186,35 +186,35 @@ function Counter() {
...
 
 </Pitfall>
 
 ---
 
-### Rendering a page partially built with React {/*rendering-a-page-partially-built-with-react*/}
+### React দিয়ে আংশিকভাবে বানানো পেইজের রেন্ডারিং {/*rendering-a-page-partially-built-with-react*/}
 
-If your page [isn't fully built with React](/learn/add-react-to-an-existing-project#using-react-for-a-part-of-your-existing-page), you can call `createRoot` multiple times to create a root for each top-level piece of UI managed by React. You can display different content in each root by calling [`root.render`.](#root-render)
+যদি আপনার পেইজ [সম্পূর্ণরূপে React দিয়ে বানানো না হয়ে থাকে](/learn/add-react-to-an-existing-project#using-react-for-a-part-of-your-existing-page), React দিয়ে পরিচালিত প্রতিটি উচ্চ স্তরের UI piece এর জন্য একটি রুট তৈরি করার জন্য আপনি একাধিকবার   you can call `createRoot` multiple times to create a root for each top-level piece of UI managed by React. You can display different content in each root by calling [`root.render`.](#root-render)
```
Comment (moshfiqrony) [inline src/content/reference/react-dom/client/createRoot.md:215]: Bhai ei line ta ki intentionally partially translate korechen?

**Review by moshfiqrony — CHANGES_REQUESTED** [review]
The translation is legit. I had some confusion so asked about it.

File: src/content/reference/react-dom/client/createRoot.md (hunk 3 above)
Comment (nafistiham) [reply src/content/reference/react-dom/client/createRoot.md:215]: Na vai miss hoye geche. Ami ekbar upor theke ekbar nich theke kori to. Ajke revise dewa hoy nai, thik kore commit dicchi jegulo bad geche.

Comment (nafistiham) [reply src/content/reference/react-dom/client/createRoot.md]: @moshfiqrony vai, all half done translations are done now.

**Pull request merged**